.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This `.vs` folder is not related with the project. Can you add to `.gitignore` and remove from the commit?

Reply:
Oh, it was visual studio files and my fault, I forget to remove them.